tdrs-frontend/cypress/support/commands.js

```diff
@@ -78,9 +78,10 @@ Cypress.Commands.add(
       method,
       body,
       url: `${Cypress.env('adminUrl')}${path}`,
+      //csrfmiddlewaretoken: ,
       form: true,
       headers: {
-        Referer: `${Cypress.env('adminUrl')}`,
+        Referer: `${Cypress.env('adminUrl')}${path}`,
```

Comment:
This is likely not needed by more closely mimics what I see via dev tools in Firefox.